lib/tasks/generate.rake

```diff
@@ -16,13 +16,22 @@ end
 def download_from_ontohub_meta(source_file, target_file)
   meta_repository
   user = User.where(admin: true).first
-  filepath = "#{Rails.root}/spec/fixtures/ontologies/#{target_file}"
-  system("wget -O #{filepath} https://ontohub.org/repositories/meta/master/download/#{source_file}")
+  filepath = "#{Rails.root}/spec/fixtures/seeds/#{target_file}"
   basename = File.basename(filepath)
+  if ENV['DOWNLOAD_FIXTURES']
+    source_url =
+      "https://ontohub.org/repositories/meta/master/download/#{source_file}"
+    temp_file = Tempfile.new([target_file, File.extname(target_file)]).path
+    if system("wget -O #{temp_file} #{source_url}")
+      filepath = temp_file
+    else
+      puts
+        'No connection to ontohub.org. Using local file for the current task.'
```

Comment:
That won't put the string, but actually prints an empty line (`puts` can take zero arguments).
Either put the string in the same line as `puts` or use Docstrings.